cogs/playerstats.py

```diff
@@ -141,6 +154,48 @@ def getAssists(comp):
     table = 'Assist Stats: ' + comp + '\n' + table
     return table
 
+def getInjuries(team="Arsenal"):
+    tableurl = "https://www.premierinjuries.com/injury-table.php"
+    #get table and convert to dataframe
+    header = {
+        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.75 Safari/537.36",
+        "X-Requested-With": "XMLHttpRequest"
+    }
+    playerDict = {"Player": "", "Reason": "", "FurtherDetail": "", "PotentialReturn": "", "Condition": "", "Status": ""}
```

Comment:
No need to define this here

Reply:
the playerDict? I tried defining it outside of the loop and it would result with just several copies of the last player for some reaosn.